nodeJS/apis/api_basics.md

```diff
@@ -7,9 +6,9 @@ Organizing your project this way can be beneficial because it allows your projec
 
 Frontend and backend applications usually talk to each other using JSON, which you have already encountered if you've gone through our frontend JavaScript course. So at this point, all you really need to learn is how to get your Express application to speak JSON instead of HTML. The assignment at the end of this lesson will take you through a tutorial, but essentially all you have to do is pass your information into [`res.json()`](https://expressjs.com/en/4x/api.html#res.json) instead of [`res.send()`](https://expressjs.com/en/4x/api.html#res.send) or [`res.render()`](https://expressjs.com/en/4x/api.html#res.render). How easy is that?
 
-If you think back to the organization of the routes in ["Part 4: Routes and controllers"](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes#Create_the_catalog_route_module) of the Local Library tutorial, all of our routes were set up in a `catalog` module, so to get the view for our list of books you would access `/catalog/books`. Setting the Library project up to also serve JSON would be as easy as creating a different router module called `api` and then adjusting the controllers so that `/catalog/books` would serve up the HTML list of our books and `/api/books` would serve up the same information as JSON.
+If you think back to the organization of the routes in ["Routers"](https://www.theodinproject.com/lessons/nodejs-routes#routers) of the Local Library tutorial, we grouped related routes together and extracted each group into its own file. This approach allowed us to more easily modify specific routes without impacting others.
 
-### Lesson overview
+## Lesson overview
```

Comment:
The heading level change is not needed.
```suggestion
### Lesson overview
```

Reply:
can I dm you on discord? I'm struggling with the PR. I am doing it for learning purpose for GIT lesson and don't want to give you a problem haha

Reply:
You can comment here in the PR so everything's in one place.

Currently, the only issues remaining are the emphasis style lint errors, where `* *` got changed to `_ _`. Our style guide needs them to be changed back to `* *`.
You can change all of these back, then just make sure you save without formatting, as it's probably your auto-formatter that's forcing them to change to `_ _` since it follows a different rule set.

Reply:
okay I think everything is good now, thank's for helping me to achieve first contribution!

Reply:
Congrats on your first PR here. Just to reiterate, in the future, don't open a PR until you've confirmed assignment by a maintainer, which is part of our contributing guide rules, and is standard practice for OSS contribution.